kubectl-cloudflow/domain/cloudflowapplication.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 
 	v1 "k8s.io/api/core/v1"
-	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
could you fix this also in some other places like the docker unit tests?

Reply:
Fixed.